packages/document/docs/en/plugin/official-plugins/client-redirects.mdx

```diff
@@ -0,0 +1,119 @@
+# @rspress/plugin-client-redirects
+
+import { SourceCode, PackageManagerTabs } from 'rspress/theme';
+
+<SourceCode href="https://github.com/web-infra-dev/rspress/tree/main/packages/plugin-client-redirects" />
+
+Used for client redirects.
+
+::: warning NOTE
+
+For this plugin to work, you need to set the fallback related configuration to the `404.html` page in the build artifacts in the relevant deployment configuration.
+
+For example:
+
+```js title="nginx"
```

Comment:
It is not recommended to use code blocks in the warning container, the UI is really ugly 😂

Maybe we can just make it as a section, such as `## Precautions`

![image](https://github.com/web-infra-dev/rspress/assets/7237365/218df643-efc4-4bfc-b62f-75f4de5df1d8)

Reply:
done